# Goal
uint8[] audio_data
---
# Result
string detected_emotion
---
# Feedback
string current_status
